docs/Developers/Newcomers_Guide.rst

```diff
@@ -210,6 +210,20 @@ Now you can start working on it.
     Before starting to write your first commit, check out this
     link: `Writing good commits <http://coala.io/commit>`_.
 
+.. seealso::
+
+    An important part of working on issues is documenting your work
+    in such a way that it is easy for others to read and understand.
+    A lot of Newcomer issues involve improving documentation.
+
+    * For more information about writing good documentation,
+      please check the following link: `Writing Documentation <https://api.coala.io/en/latest/Developers/Writing_Documentation.html>`_
+
+    * For more information about how to style Python code
+      according to the PEP8 code style convention,
```

Comment:
I think `style` is more suitable over here as official docs of PEP8 also refers it as `style`. https://www.python.org/dev/peps/pep-0008